command/build.py

```diff
@@ -1,8 +1,32 @@
-class BuildCmd():
-    name = 'build'
+import yaml
+import command.identify as identify
```

Comment:
how about `from command import identify` ?

Reply:
addressed in latest commit